packages/hardhat-core/src/internal/solidity/compiler/downloader.ts

```diff
@@ -126,7 +126,10 @@ export class CompilerDownloader implements ICompilerDownloader {
   }
 
   public static defaultCompilerListCachePeriod = 3_600_00;
-  private readonly _mutex = new Mutex();
+  private readonly _compilerDownloadMultiProcessMutex = new MultiProcessMutex(
```

Comment:
I think `_mutex` was fine. The `MultiProcess` part is a implementation information, it doesn't need to be in the variable name. And since this is the only mutex in the compiler downloader module, then `compilerDownload` can go too.

Reply:
Done